{"name": "test_login_3[3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_fixture.TestLogin object at 0x000002C7D28F9358>, data = 3\n\n    def test_login_3(self, data):\n        \"\"\"记住密码登录\"\"\"\n        allure.attach(\"输入用户名\", \"输入用户名username_3\")\n        print(\"输入用户名\")\n        allure.attach(\"输入密码\", \"输入用户名password_3\")\n        print(\"输入密码\")\n        allure.attach(\"点击登录\", \"登录成功\")\n        print(\"点击登录\")\n        print(data)\n>       assert False\nE       assert False\n\nscripts\\test_fixture.py:40: AssertionError"}, "description": "记住密码登录", "attachments": [{"name": "输入用户名username_3", "source": "83dd1a33-23eb-4ed0-a82b-ee4061db23c3-attachment.attach"}, {"name": "输入用户名password_3", "source": "cab115ac-4ec3-4ec5-8d2b-a97d38563761-attachment.attach"}, {"name": "登录成功", "source": "24ca7ba4-67ad-4af1-a1d1-c91d271cff68-attachment.attach"}], "parameters": [{"name": "data", "value": "3"}], "start": 1575025966257, "stop": 1575025966258, "uuid": "a29c8094-58cf-47f4-80da-5b125f4efae6", "historyId": "c19157f3bc16370952e1078628fc2183", "testCaseId": "133a8e3ccd283ba22e290157fe855949", "fullName": "scripts.test_fixture.TestLogin#test_login_3", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47A72RD"}, {"name": "thread", "value": "6108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture"}]}